delegate: add schema to SHOW SEQUENCES

otan opened the pull request:
Resolves #54962 .

Release note (sql change): SHOW SEQUENCES now show sequences in user
defined schemas. The schema is displayed in a newly added
`sequence_schema` column.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/55083)
<!-- Reviewable:end -->

otan committed:
delegate: add schema to SHOW SEQUENCES

Release note (sql change): SHOW SEQUENCES now show sequences in user
defined schemas. The schema is displayed in a newly added
`sequence_schema` column.

arulajmani reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/55083#-:-MIb2hEL2LFn_eeZ59HT:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @arulajmani)



<!-- Sent from Reviewable.io -->

otan commented:
bors r=arulajmani
